File: TestArgent/TestArgent/StandMenager/AspirateurQuantique.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace TestArgent
{
    class AspirateurQuantique : Usine
    {
        public AspirateurQuantique()
        {
            niveau = 0;
            this.gainArgent =100;
            this.tempsDeProduction = 30;
            this.nom = "Aspirateur Quantique(";
            this.coutAmelioration = 1000;
            acheter = false;
            prerequis = "CPU Ameliorer NIV 15";
            numUsine = 5;
        }
    }
}

File: Projet1/Tycoon/AspirateurQuantique.cpp
#include "AspirateurQuantique.h"

AspirateurQuantique::AspirateurQuantique() {
    niveau = 0;
    gainArgent = 100;
    tempsDeProduction = 30;
    nom = "Aspirateur Quantique(";
    coutAmelioration = 1000;
    acheter = false;
    prerequis = "CPU Ameliorer NIV 15";
    numUsine = 5;
}